charmcraft/templates/init/metadata.yaml.j2

```diff
@@ -5,5 +5,18 @@ description: |
   TEMPLATE-TODO: fill out the charm's description
 summary: |
   TEMPLATE-TODO: fill out the charm's summary
-series: {{ series }}
+bases:
+  - name: ubuntu
+    channel: 20.04/stable
```

Comment:
Should we add a "TEMPLATE-TODO" around these values? or as they are for the charm (and not the "payload") it's just fine to leave them as is?

Reply:
I would have said fine to leave as they are sensible defaults for the charm, but I've removed `bases` entirely now :)